tds-itemselection-impl/src/main/java/tds/itemselection/impl/sets/ItemPool.java

```diff
@@ -334,6 +334,7 @@ private Double float2Double(DbResultRecord record, String columnName)
     {
     	return this._items.values();
     }
+    public Collection<TestItem> getSiblingItems() { return this._siblingItems.values(); }
```

Comment:
Please add a line break between the methods and let's put the `return` on its own line.  That is normal convention in most of the Java development I've done.